src/geocat/viz/util.py

```diff
@@ -1240,19 +1441,19 @@ def set_vector_density(data: xr.DataArray,
         return ds
 
 
-def get_skewt_vars(p: Quantity, tc: Quantity, tdc: Quantity,
-                   pro: Quantity) -> str:
+def get_skewt_vars(pressure: Quantity, temp_parcel: Quantity,
```

Comment:
For the parameters here we might look to metpy: https://unidata.github.io/MetPy/latest/api/generated/metpy.calc.cape_cin.html

Reply:
good call! done